<?xml version="1.0" encoding="UTF-8"?><testrun name="H__Networking_Prog0Test.jar (Failed Tests first)" project="FoodItem" tests="65" started="65" failures="15" errors="0" ignored="0">
  <testsuite name="foodnetwork.serialization.donatest.FoodItemTest" time="0.016">
    <testsuite name="[2]" time="0.01">
      <testcase name="testShortEncode[2]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.004">
        <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.junit.runners.Suite.runChild(Suite.java:128)&#13;
	at org.junit.runners.Suite.runChild(Suite.java:27)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
      </testcase>
      <testcase name="testGetSetName[2]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testGetSetMealType[2]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testGetSetCalories[2]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testDecode[2]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.001"/>
      <testcase name="testEncode[2]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.004"/>
      <testcase name="testGetSetFat[2]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testDoubleDecode[2]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testDoubleEncode[2]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testConstructor[2]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
    </testsuite>
    <testsuite name="[1]" time="0.003">
      <testcase name="testShortEncode[1]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.001">
        <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.junit.runners.Suite.runChild(Suite.java:128)&#13;
	at org.junit.runners.Suite.runChild(Suite.java:27)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
      </testcase>
      <testcase name="testGetSetName[1]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testGetSetMealType[1]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testGetSetCalories[1]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testDecode[1]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.001"/>
      <testcase name="testEncode[1]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testGetSetFat[1]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.001"/>
      <testcase name="testDoubleDecode[1]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testDoubleEncode[1]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testConstructor[1]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
    </testsuite>
    <testsuite name="[0]" time="0.003">
      <testcase name="testShortEncode[0]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.001">
        <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.junit.runners.Suite.runChild(Suite.java:128)&#13;
	at org.junit.runners.Suite.runChild(Suite.java:27)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
      </testcase>
      <testcase name="testGetSetName[0]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testGetSetMealType[0]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testGetSetCalories[0]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testDecode[0]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testEncode[0]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testGetSetFat[0]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testDoubleDecode[0]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.001"/>
      <testcase name="testDoubleEncode[0]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.0"/>
      <testcase name="testConstructor[0]" classname="foodnetwork.serialization.donatest.FoodItemTest" time="0.001"/>
    </testsuite>
  </testsuite>
  <testsuite name="foodnetwork.serialization.donatest.MealTypeTest" time="0.001">
    <testcase name="testGetMealTypeBadCode" classname="foodnetwork.serialization.donatest.MealTypeTest" time="0.0"/>
    <testcase name="testGetMealType" classname="foodnetwork.serialization.donatest.MealTypeTest" time="0.0"/>
    <testcase name="testGetMealTypeCode" classname="foodnetwork.serialization.donatest.MealTypeTest" time="0.001"/>
  </testsuite>
  <testsuite name="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.01">
    <testcase name="testLetterFat" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0">
      <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testLongMealType" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0">
      <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testLetterCaloriesDecode" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.001">
      <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testNullFatSet" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0">
      <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testNegativeCaloriesSet" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0">
      <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testPoorlyFormattedFat" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0">
      <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testBadMealType" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0">
      <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testNullMealTypeSet" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0">
      <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testPoorlyFormattedCalories" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0">
      <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testNameLengthDecode" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0">
      <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testNegativeCaloriesConstructor" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0">
      <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testEmptyNameSet" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0">
      <failure>java.lang.AssertionError: Expected exception: foodnetwork.serialization.FoodNetworkException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testBorderCalories" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.001"/>
    <testcase name="testNullNameConstructor" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
    <testcase name="testNullNameSet" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
    <testcase name="testNullFatConstructor" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
    <testcase name="testHashcodeEqualsObjectsSameNumber" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
    <testcase name="testHashcodeConsistent" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
    <testcase name="testEqualsReflexive" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
    <testcase name="testEqualsNull" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
    <testcase name="testEqualsWrongType" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.001"/>
    <testcase name="testEqualsSymmetric" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.002"/>
    <testcase name="testParallelDecode" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.002"/>
    <testcase name="testParallelEncode" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
    <testcase name="testNullMealTypeConstructor" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.001"/>
    <testcase name="testBorderFat" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
    <testcase name="testEqualsTransitive" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
    <testcase name="testEmptyNameConstructor" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
    <testcase name="testParallelConstruction" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
    <testcase name="testEqualsConsistent" classname="foodnetwork.serialization.donatest.FoodItemBoundsTest" time="0.0"/>
  </testsuite>
  <testsuite name="foodnetwork.serialization.donatest.FoodNetworkExceptionTest" time="0.001">
    <testcase name="testAddATudeExceptionConstructorCause" classname="foodnetwork.serialization.donatest.FoodNetworkExceptionTest" time="0.0"/>
    <testcase name="testAddATudeExceptionConstructor" classname="foodnetwork.serialization.donatest.FoodNetworkExceptionTest" time="0.001"/>
  </testsuite>
</testrun>
